lib/opensusebasetest.pm

```diff
@@ -651,7 +651,7 @@ sub grub_select {
         push @tags, 'linux-login' if check_var('DESKTOP', 'textmode');
         push @tags, 'displaymanager' if check_var('DESKTOP', 'gnome');
 
-        assert_screen(\@tags);
+        assert_screen(\@tags, timeout => 1000);
```

Comment:
Why we need to change this? Is this only applicable for ppc?

Reply:
Good point, due to current situation, we have to run ppc64le image osd test of SL Mircro6.1 via QEMU without KVM, mean that there is low performance during test run. At current DEBUG step, try to use TIME OUT way to finish all test modules for ppc64le images test run.